samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/Controllers/AuthController.cs

```diff
@@ -0,0 +1,267 @@
+﻿// <copyright file="AuthController.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+using Azure.Core;
+using MeetingTranscriptRecording.Helper;
+using MeetingTranscriptRecording.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Graph;
+using Newtonsoft.Json;
+using System.Net.Http.Headers;
+using System.Net;
+
+namespace MeetingTranscriptRecording.Controllers
+{
+    public class AuthController : Controller
+    {
+        /// <summary>
+        /// Represents a set of key/value application configuration properties.
+        /// </summary>
+        public readonly IConfiguration _configuration;
+        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public AuthController(
+            IConfiguration configuration,
+            IHttpClientFactory httpClientFactory,
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _configuration = configuration;
+            _httpClientFactory = httpClientFactory;
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        /// <summary>
+        /// This method retrieves information about the event details.
+        /// It obtains an access token and makes requests to the Microsoft Graph API to fetch events, online meetings, transcripts, and recordings.
+        /// It constructs a JSON response with data related to online meetings.
+        /// Error handling is included in case of exceptions.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("GetEventInformation")]
+        public async Task<JsonResult> GetEventInformation()
+        {
+            try
+            {
+                var CardResults = new List<CardData>();
+
+                var eventCount = 0;
+
+                var accessToken = await AuthHelper.GetAccessTokenOnBehalfUserAsync(_configuration, _httpClientFactory, _httpContextAccessor);
+
```

Comment:
this url should be come from appsetting. add "https://graph.microsoft.com/beta" as base url in appsetting and concatinate here

Reply:
Fixed